testcases/commands/unzip/unzip01.sh

```diff
@@ -52,6 +52,27 @@ Archive:  $TST_DATAROOT/test.zip
    creating: dir/d3/d2/
    creating: dir/d3/d3/
 EOF
+
+	cat > unzip_exp02.out <<EOF
```

Comment:
this looks like you're just picking another random unzip program.

i'm not sure we should even bother keeping unzip tests.  they don't seem that useful.  this will need discussion on the mailing list.